server/tests/security/test_credential_creation.py

```
@@ -0,0 +1,143 @@
+    def test_user_management_blocked(self, cmd):
+        assert deny_blocks(cmd), f"User management not denied: {cmd!r}"
+
+
+# ---------------------------------------------------------------------------
+# 4. Credential file access
+# ---------------------------------------------------------------------------
+
+class TestCredentialFileAccess:
+    """Reading sensitive credential files must be caught by the signature matcher."""
+
+    @pytest.mark.parametrize("cmd,expected_rule", [
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)